d3a_api_client/commands.py

```diff
@@ -16,12 +19,18 @@ def offer_energy(self, area_uuid, energy, price):
     def offer_energy_rate(self, area_uuid, energy, rate):
         return self._add_to_buffer(area_uuid, Commands.OFFER, {"energy": energy, "rate": rate})
 
+    def update_offer(self, area_uuid, energy, price):
+        return self._add_to_buffer(area_uuid, "update_offer", {"energy": energy, "price": price})
```

Comment:
minor, but you could add update_offer and update_bid to the `Offers` class.

Reply:
@hannesdiedrich  You mean to create an Enum for storing these string values ?

Reply:
@hannesdiedrich If I understood correctly I should create an Enum class for these String values ?

Reply:
@Hassan754 is is already there https://github.com/gridsingularity/d3a-api-client/blob/fc5f01facac3c9f0f297ef9921b3387817f7114c/d3a_api_client/enums.py#L4 please add the "new" commands

Reply:
Sorry, I just saw that I miswrote in my first comment I meant the `Commands` class of course.

Reply:
@hannesdiedrich This is not possible because the Commands Enum class is consumed the `RedisClientBase` , it is iterated and subscribed to all commands in it.   d3a_api_client/redis_client_base.py:48
Some of them are common channels name so this will throw error
If we're to add these values to an Enum, I can create a specific enum which inherits from the CommandsENum and extends as much as needed

Reply:
@Hassan754 `update_offer` and `update_bid` are also commands that should be available for the redis client. 
So I see no harm in that. You will need to adapt `RedisClient._response_topics` then.
I dont know how urgent this PR here, but if it is really important you could go and create an enhancement story for after the deploy that homoginzes these command enums.

Reply:
@hannesdiedrich Actually it's quiet urgent, Sarah is constantly pinging me , anyway I created a separate ticket https://gridsingularity.atlassian.net/browse/D3ASIM-3319